source/administration/upgrade.rst

```diff
@@ -1,120 +1,75 @@
+
+.. csv-table::
+  :header: "If your current version is:", "Read these Important Notices:", "Use these instructions:"
+  :widths: auto
+
+  "3.9.0", "Not Applicable", "`Upgrading to the latest version <../administration/upgrading-to-latest.html>`_"
+  "3.8.x", "See `Important Notices`_ 7", "`Upgrading to the latest version <../administration/upgrading-to-latest.html>`_"
+  "3.7.x", "See `Important Notices`_ 4 to 7", "`Upgrading to the latest version <../administration/upgrading-to-latest.html>`_"
+  "3.6.x", "See `Important Notices`_ 4 to 7","`Upgrading to the latest version <../administration/upgrading-to-latest.html>`_"
+  "3.5.x", "See `Important Notices`_ 2 to 7","`Upgrading to the latest version <../administration/upgrading-to-latest.html>`_"
+  "3.4.0", "See `Important Notices`_ 2 to 7","`Upgrading to the latest version <../administration/upgrading-to-latest.html>`_"
+  "3.3.0", "See `Important Notices`_ 1 to 7","`Upgrading to the latest version <../administration/upgrading-to-latest.html>`_"
+  "3.2.0", "See `Important Notices`_ 1 to 7","`Upgrading to the latest version <../administration/upgrading-to-latest.html>`_"
+  "3.1.0", "See `Important Notices`_ 1 to 7","`Upgrading to the latest version <../administration/upgrading-to-latest.html>`_"
+  "3.0.3", "See `Important Notices`_ 1 to 7","`Upgrading to the latest version <../administration/upgrading-to-latest.html>`_"
+  "2.2.0", "Not Applicable", "`Upgrading to version 3.0 <../administration/upgrading-to-3.0.html>`_"
+  "2.1.0", "Not Applicable", "`Upgrading to version 3.0 <../administration/upgrading-to-3.0.html>`_"
+  "2.0.0", "Not Applicable", "`Upgrading to version 3.0 <../administration/upgrading-to-3.0.html>`_"
+  "1.4.0", "Not Applicable", "`Upgrading to version 2.0 <../administration/upgrading-to-2.0.html>`_"
+  "1.3.0", "Not Applicable", "`Upgrading to version 2.0 <../administration/upgrading-to-2.0.html>`_"
+  "1.2.1", "Not Applicable", "`Upgrading to version 2.0 <../administration/upgrading-to-2.0.html>`_"
+  "1.2.0", "Not Applicable", "`Upgrading to version 2.0 <../administration/upgrading-to-2.0.html>`_"
+  "1.1.1", "Not Applicable", "`Upgrading to version 2.0 <../administration/upgrading-to-2.0.html>`_"
+  "1.1.0", "Not Applicable", "`Upgrading to version 2.0 <../administration/upgrading-to-2.0.html>`_"
+  "1.0.0", "Not Applicable", "`Upgrading to version 2.0 <../administration/upgrading-to-2.0.html>`_"
+
+.. toctree::
+   :maxdepth: 1
+
+   /administration/upgrading-to-latest.rst
+   /administration/upgrading-to-3.0.rst
+   /administration/upgrading-to-2.0.rst
 
 Important Notices
 -----------------
 
-1. Security related changes were made in 3.9.0 that cause any previously created team invite links, password reset links, and email verification links to no longer work. You must update any place where you have published these links.
```

Comment:
"After upgrading to version 3.4" > I think this means version 3.4 and later, from any version before 3.4 (need to figure out wording). I think we need to be explicit in describing when people will hit the notes